linkerd/docs/routers.md

```diff
@@ -207,3 +200,82 @@ minSize | `0` | The minimum number of connections to maintain to each host.
 maxWaiters | Int.MaxValue | The maximum number of connection requests that are queued when the connection concurrency exceeds maxSize.
+
+<a name="service-configuration"></a>
+## Service Configuration
+
+This section defines the policy that linkerd will use when talking to services.
+The structure of this section depends on its `kind`.
+
+Key  | Default Value    | Description
+---- | ---------------- | --------------
+kind | `io.l5d.global` | Either [io.l5d.global](#global-service-configuration) or [io.l5d.static](#static-service-configuration)
+
+<a name="global-service-configuration"></a>
+### Global Service Config
+
+```yaml
+- protocol: http
+  service:
+    kind: io.l5d.global
+    totalTimeoutMs: 500ms
+    retries:
+      budget:
+        minRetriesPerSec: 5
+        percentCanRetry: 0.5
+        ttlSecs: 15
+      backoff:
+        kind: jittered
+        minMs: 10
+        maxMs: 10000
+```
+
+This service configuration allows you to specify [service parameters](#service-parameters)
+which will be applied to all services.
+
+<a name="static-service-configuration"></a>
+### Static Service Config
+
+```yaml
+- protocol: http
+  service:
+    kind: io.l5d.static
+    configs:
+    - prefix: /svc
+      retries:
+        budget:
+          minRetriesPerSec: 5
+          percentCanRetry: 0.5
+          ttlSecs: 15
+        backoff:
+          kind: jittered
+          minMs: 10
+          maxMs: 10000
+    - prefix: /svc/foo
+        totalTimeoutMs: 500ms
+    - prefix: /svc/bar
+        totalTimeoutMs: 200ms
+
+```
+
+This service configuration allows you to specify [service parameters](#service-parameters)
+which will be applied to all services that match a specified prefix.  The service
+configuration must contain a property called `configs` which contains a list
+of config objects.  Each config object must specify a prefix and the
+[service parameters](#service-parameters) to apply to services that match that prefix.
+If a service matches more than one config's prefix, all parameters from the
+matching configs will be applied, with parameters from later configs taking
+precedence.
+
+<a name="service-parameters"></a>
+### Service Parameters
+
+<aside class="notice">
+These parameters are available to the service regardless of protocol. Services may also have protocol-specific parameters.
+</aside>
+
+Key                 | Default Value            | Description
+------------------- | -------------------------| -----------
+retries             | see [retries](#retries)  | A [retry policy](#retries) for applicaiton-level retries.
+totalTimeoutMs      | no timeout               | The timeout for an entire request, including all retries, in milliseconds.
+responseClassifier  | `io.l5d.nonRetryable5XX` | A (sometimes protocol-specific) [response classifier](#http-response-classifiers) that determines which responses should be considered failures and, of those, which should be considered [retryable](#retries).
```

Comment:
I think we need to explain how this setting influences server and client modules.